intranet/apps/eighth/models.py

```diff
@@ -1248,11 +1268,13 @@ def remove_signup(self, user=None, force=False, dont_run_waitlist=False):
             return "Successfully removed signup for {}.".format(block)
 
     def accept_pass(self):
+        """Sets was_absent=False, pass_accepted=True, and saves both fields."""
         self.was_absent = False
         self.pass_accepted = True
         self.save(update_fields=["was_absent", "pass_accepted"])
 
     def reject_pass(self):
+        """Sets was_absent=True, pass_accepted=True, and saves both fields."""
```

Comment:
Probably could become:
```suggestion
        """Rejects an eighth period pass for the EighthSignup object."""
```